src/models/data.cpp

```diff
@@ -353,3 +353,61 @@ const Data::ThreadEvents* Data::EventResults::findThread(qint32 pid, qint32 tid)
 {
     return const_cast<Data::EventResults*>(this)->findThread(pid, tid);
 }
+
+DiffViewResults DiffViewResults::fromTopDown(const TopDownResults& a, const TopDownResults& b)
+{
+    DiffViewResults results;
+    results.costs.initializeCostsFrom(a.selfCosts);
+    results.costs.setTotalCosts(QVector<qint64>(results.costs.numTypes(), 1'000'000));
```

Comment:
I used the self cost of a symbol to calculate the difference in percent. Since costs uses an int to store the cost I used a somewhat big number to get some decimal places. Is this stupid or a good idea?